projects/npm-tools/packages/npm-scripts/src/utils/createFederationConfig.js

```diff
@@ -189,6 +190,56 @@ module.exports = async function () {
 	remotes = remotes || [];
 	shared = shared || [];
 
+	// Prevent webpack from choking on scopes
+
+	const libraryName = makeNamespace(name);
```

Comment:
If we don't namespace the library name here webpack gets fooled by the `@` and probably the `/`.

We could try to get it fixed but since webpack uses `@` for other purposes in the context of federation, it would be a continuous source of trouble, I'm afraid.